recipes/umi-transfer/build.sh

```diff
@@ -1,10 +1,15 @@
-#!/bin/bash -euo
+#!/bin/bash -euo pipefail
 export CFLAGS="${CFLAGS} -fcommon"
-export CXXFLAGS="${CFLAGS} -fcommon"
+export CXXFLAGS="${CXXFLAGS} -fcommon"
+export RUSTFLAGS="-C link-arg=-s"
 
 # Add workaround for SSH-based Git connections from Rust/cargo.  See https://github.com/rust-lang/cargo/issues/2078 for details.
 # We set CARGO_HOME because we don't pass on HOME to conda-build, thus rendering the default "${HOME}/.cargo" defunct.
 export CARGO_NET_GIT_FETCH_WITH_CLI=true CARGO_HOME="$(pwd)/.cargo"
 
 # build statically linked binary with Rust
-RUST_BACKTRACE=1 cargo install --verbose --root $PREFIX --path .
\ No newline at end of file
+RUST_BACKTRACE=1 cargo install --verbose --locked --root $PREFIX --path .
+
+if command -v ldd &> /dev/null; then
+    echo $(ldd $PREFIX/bin/umi-transfer)
```

Comment:
This debug statement could be removed

Reply:
I wanted to check if the specified dependencies are truly needed or not. It seems that zlib and the C++ compiler might not be needed.